docs/guidance/model/organizational_units.md

```diff
@@ -0,0 +1,99 @@
+
+## Worked examples
+
+### 1. Using the Organization building block
+
+In Honduras, the Ministry of Health is planning the procurement of food supplies for the San Felipe Hospital. For the purposes of the example, San Felipe Hospital is considered to be a unit belonging to the Ministry of Health, and it is not a legal entity of its own.
+
+In the release below, the publisher adds the hospital name at the end of the procuring entity name. The main identifier for the organization (*Secretaría de Salud Pública*) is extracted from a local list in the "HonduCompras" platform, used to publish procurement information in the country.
+
+An identifier for the hospital has been added using the "X-HN-ONCAE-UNIT" list code. The `address` and `contactPoint` information belongs to the hospital only.
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/organizational-units/honduras-planning.json
+   :jsonpointer: 
+   :expand: releases, parties, identifier, additionalIdentifiers
+   :title: release
+
+```
+
+### 2. Defining a new Extension
+
+In Moldova, the national procurement agency needs to include a division code for particular organizations. Since divisions can be separate legal entities in some cases, the publisher chooses to use the `identifier` block to point to the main organization for all cases, and use an additional field to provide the division code that leads data users to locate the departments and branches involved.
+
+In the release that follows, the Bank of Moldova announces a contract opportunity for the provision of consumables for electrical appliances.
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/organizational-units/moldova-tender.json
+   :jsonpointer: 
+   :expand: releases, parties, identifier, additionalIdentifiers, details
+   :title: release
+
+```
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/organizational-units/division-code-extension/extension.json
+   :jsonpointer: 
+   :expand: name, description, schemas, compatibility
+   :title: extension
+
+```
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/organizational-units/division-code-extension/release-schema.json
+   :jsonpointer: 
+   :expand: definitions, Organization, properties, details, properties, divisionCode
+   :title: release-schema
+
+```
+
+The branch name (*Chişinău Branch*) is appended at the end of the name of the procuring entity. A new extension called "Division code" has been defined to add the `divisionCode` field in the `parties/details` section. The branch code in the example is "101".
+
+The `extension.json` and `release-schema.json` files for the Division code extension can be displayed using the combo box above the json example.
```

Comment:
'json' -> 'JSON' for consistency?

Also, do we need to explain how the display widget works? Maybe just say that the extension and release-schema are also given in the example above, or move this sentence to above the example (line 41) and say "The `extension.json` and `release-schema.json` files for the Division code extension are also given below"?

Reply:
To be honest, it took me a while to figure that there was a combobox at the top of the JSON files in the documentation. So I guess I fear someone may have not notice that as well!

Reply:
Yes, good point. Maybe we need to look at the widget again to make it clear (I agree it's not obvious!) or some top-level guidance for the examples section?

Reply:
I believe it's best for instructions to be in place, because not everyone reads all the documentation. I agree the widget could be clearer, indicating the number of files available, perhaps?

Reply:
I created an issue to consider alternatives: #1020 1021